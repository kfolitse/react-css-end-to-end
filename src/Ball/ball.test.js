import React from 'react'
import renderer from 'react-test-renderer'
import { Ball } from '..'

it('renders correctly', () => {
  const tree = renderer.create(<Ball />).toJSON()
  expect(tree).toMatchSnapshot()
})
