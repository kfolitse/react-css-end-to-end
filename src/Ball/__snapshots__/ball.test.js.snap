// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style>
     
    [object Object]
  </style>,
  <div
    className="loadingio-spinner-ball-4t8cn6gusn7"
  >
    <div
      className="ldio-bvcqwv5eb07"
    >
      <div />
    </div>
  </div>,
]
`;
